theme/js/components/version-selector.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+const $ = global.jQuery;
+
+class VersionSelector {
+  constructor() {
+    this._getVersions(versions => {
+      this._render(versions);
+    });
+  }
+
+  get currentComponent() {
+    return window.location.pathname.split('/').slice(2).join('/') || null;
+  }
+
+  get currentVersion() {
+    return window.location.pathname.split('/')[1] || '';
+  }
+
+  toggleSelector() {
+    const button = $('#version_selector_button');
+    const dropdownMenu = $('#version_selector_dropdown_menu');
+    if (button.attr('aria-expanded') === 'true') {
+      dropdownMenu.hide();
+      button.attr('aria-expanded', 'false');
+    }
+    else {
+      dropdownMenu.show();
+      button.attr('aria-expanded', 'true');
+    }
+  }
+
+  _getVersions(callback) {
+    const versionRequest = new XMLHttpRequest();
+    versionRequest.addEventListener('load', function () {
+      const jsonVersions = JSON.parse(this.responseText);
+
+      const versions = jsonVersions.versions.reduce(
+        function (total, v) {
+          if (!v.branch) {
+            total.releases = [v].concat(total.releases).slice(0, 5);
+          } else if (v.branch === 'topic') {
+            total.topicBranches.push(v);
+          }
+
+          return total;
+        },
+        { releases: [], topicBranches: [] }
+      );
+
+      callback(versions);
+    });
+    versionRequest.open('GET', '/versions.json?t=' + new Date().getTime());
+    versionRequest.send();
+  }
+
+  _render(versions) {
+    const container = $('.toolkit-version-selector');
+
+    const createListItem = version => {
+      const li = $('<li/>');
+      const a = $('<a/>');
+      li.append(a);
+      a.attr('href', `/${version.version}${this.currentComponent ? '/' + this.currentComponent : ''}`);
+      a.text(version.label || version.version);
+      if (this.currentVersion === version.version) {
+        li.addClass('dso-checked');
+      }
+
+      return li;
+    };
+
+    container.prepend(
+      '<li>' +
```

Comment:
Waarom hier geen backtik / template literal gebruiken?

Reply:
En zou je deze `li` van de classes `navbar-text dropdown` willen voorzien en de `div` op de volgende regel willen verwijderen?

![image](https://user-images.githubusercontent.com/9696261/91304107-41580c00-e7a9-11ea-877b-549cb67fdac7.png)

de status modifier `open` komt dus ook op de `li`.

Ik weet dat dit (nog) niet zo expliciet in de toolkit staat omschreven, maar ik wil kijken hoever we komen.